keel-sql/src/main/resources/db/changelog/20201016-create-lifecycle-events.yml

```diff
@@ -0,0 +1,151 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-lifecycle-events
+      author: emjburns
+      changes:
+        - createTable:
+            tableName: lifecycle
+            columns:
+              - column:
+                  name: uid
+                  type: char(26)
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: scope
+                  type: varchar(50)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: ref
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
```

Comment:
Would it make sense to model this as a foreign key to `delivery_artifact`?

Reply:
I think it might! Can we chat offline about foreign keys? I haven't worked with them and wanted to ask about the benefits of modeling things using them.

Reply:
discussed offline: not doing this, because serializing artifacts from their db records requires special handling. since we're not using any fields on the artifact, I don't think this makes sense.